rules/travis.js

```diff
@@ -74,15 +75,23 @@ module.exports = ctx => {
 				});
 			}
 
-			const versions = Array.isArray(travis.node_js) ? travis.node_js : [travis.node_js];
+			let versions = Array.isArray(travis.node_js) ? travis.node_js : [travis.node_js];
+
+			versions = versions.filter(v => IGNORED_VERSIONS.indexOf(v) === -1);
 
 			for (const version of versions) {
 				if (DEPRECATED_VERSIONS.indexOf(version) !== -1) {
 					ctx.report({
 						message: `Version \`${version}\` is deprecated.`,
 						file
 					});
-				} else if (version !== 'node' && engine && !semver.satisfies(normalize(version), engine)) {
+				}
+			}
+
+			versions = versions.filter(v => DEPRECATED_VERSIONS.indexOf(v) === -1);
```

Comment:
I'm looking back at the code and wondering why we can't write the above for loop like what we had before.

```js
for (const version of versions) {
	if (DEPRECATED_VERSIONS.indexOf(version) !== -1) {
		ctx.report({
			message: `Version \`${version}\` is deprecated.`,
			file
		});
	} else if (engine && !semver.satisfies(normalize(version), engine)) {
		ctx.report({
			message: `Unsupported version \`${version}\` is being tested.`,
			fix: fixers.unsupported(version),
			file
		});
	}
}
```

I believe this does the same thing as your code with 2 for-loops without having to filter out the deprecated versions.